examples/atomic-registry/Makefile

```diff
@@ -21,8 +21,9 @@ install-atomic:
 install-systemd:
 	atomic install $(TEST_IMAGE) $(INSTALLHOST)
 	systemctl enable --now atomic-registry-master.service
-	sleep 10
-	/var/run/setup-atomic-registry.sh $(INSTALLHOST)
+	echo "Waiting for API"
+	while [[ ! $$(/bin/curl -kfLs https://localhost:8443/healthz) = "ok" ]]; do /bin/sleep 1 && echo -n "."; done
```

Comment:
You don't actually need to do string comparisons since you're using the `--fail|-f` with `curl`, the result is a little more legible:

```sh
while ! /bin/curl -kfLs https://localhost:8443/healthz; do
    /bin/sleep 1
    echo -n '.'
done
```